packages/pluggableWidgets/timeline-web/src/Timeline.editorConfig.ts

```diff
@@ -1,5 +1,13 @@
-import { hidePropertiesIn, hidePropertyIn, Problem, Properties } from "@widgets-resources/piw-utils";
+import {
+    hidePropertiesIn,
+    hidePropertyIn,
+    Problem,
+    Properties,
+    StructurePreviewProps
+} from "@widgets-resources/piw-utils";
 import { TimelinePreviewProps } from "../typings/TimelineProps";
+import { lineAndDotSVG } from "./preview/preview";
+import { getHeaderOption, GroupHeaderConfig } from "./components/TimelineComponent";
```

Comment:
Should be extracted in a new file.. like `utils.ts`

Reply:
👍